app/containers/GameBoard.jsx

```diff
@@ -162,7 +157,7 @@ class GameBoard extends Component {
           </div>
           <div id="btns">
             <button data-testid="new-game" onClick={this.newGame}>New Game</button>
-            <button data-testid="battle" onClick={() => this.battle(this.state.decks)}>Battle</button>
+            <button data-testid="battle" onClick={this.battle}>Battle</button>
```

Comment:
since this.state is available to your function, we don't want to parameterize it.